src/App.js

```diff
@@ -1,23 +1,186 @@
-import logo from './logo.svg';
-import './App.css';
+import { useEffect, useState } from 'react';
+import './App.scss';
+import ship from './assets/ship.png';
+
```

Comment:
Not sure if you would maybe want to remove some of the blank spaces between blocks of code like the one here in between your asset imports, just to save a couple lines here and there